app/models/pglogical_subscription.rb

```diff
@@ -86,6 +86,16 @@ def pglogical(refresh = false)
     self.class.pglogical(refresh)
   end
 
+  def validate(some_connection_params = {})
```

Comment:
This params hash could probably have a better name. Maybe `new_connection_options` or something similar?